src/ui/controllers/user.ts

```diff
@@ -193,147 +194,6 @@ export function parseRoles(traversonResult: any): [Level[], string] | void {
 	}
 }
 
-export async function newRenderAreasOfWorkPage(
```

Comment:
can you explain why this function and one below are removed? are they not referenced?